README.md

```diff
@@ -1,39 +1,99 @@
-![Build Status](https://azure-iot-sdks.visualstudio.com/azure-iot-sdks/_apis/build/status/Azure.azure-iot-sdk-python)
-
 # Azure IoT Hub Python SDKs v2 - PREVIEW
 
-This repository contains the code for the future v2.0.0 of the Azure IoT SDKs for Python. The goal of v2.0.0 is to be a complete rewrite of the existing SDK that maximizes the use of the Python language and its standard features rather than wrap over the C SDK, like v1.x.x of the SDK did.
+![Build Status](https://azure-iot-sdks.visualstudio.com/azure-iot-sdks/_apis/build/status/Azure.azure-iot-sdk-python)
+
+This repository contains code for the Azure IoT SDKs for Python.  This enables python developers to easily create IoT devices that work semealessly 
 
 *If you're looking for the v1.x.x client library, it is now preserved in the [v1-deprecated](https://github.com/Azure/azure-iot-sdk-python/tree/v1-deprecated) branch.*
 
 **Note that these SDKs are currently in preview, and are subject to change.**
 
-# SDKs
+## Python SDK
 
-This repository contains the following SDKs:
+This repository contains the following:
 
-* [Azure IoT Device SDK](azure-iot-device) - /azure-iot-device
-    * Provision a device using the Device Provisioning Service for use with the Azure IoT hub
-    * Send/receive telemetry between a device or module and the Azure IoT hub or Azure IoT Edge device
-    * Handle direct methods invoked by the Azure IoT hub on a device
-    * Handle twin events and report twin updates
-    * *Still in development*
-        - *Blob/File upload*
-        - *Invoking method from a module client onto a leaf device*
+* *Azure IoT Device SDK*
 
-* Azure IoT Hub SDK **(COMING SOON)**
-    * Do service/management operations on the Azure IoT Hub
+  * Send/receive telemetry between a device or module and the Azure IoT hub or Azure IoT Edge device
+  * Handle direct methods invoked by the Azure IoT hub on a device
+  * Handle twin events and report twin updates
+  * *Still in development*
+    * *Blob/File upload*
+    * *Invoking method from a module client onto a leaf device*
 
-* Azure IoT Hub Provisioning SDK **(COMING SOON)**
-    * Do service/management operations on the Azure IoT Device Provisioning Service
+* Azure IoT Hub SDK
+  * Do service/management operations on the Azure IoT Hub
+  * *Still in development*
+    * C2D messaging
+    * Direct Methods
+    * Configurations
+    * Query devices
+    * IoTHub Job
 
-# How to install the SDKs
+* Azure IoT Hub Provisioning SDK
+  * Provision a device using the Device Provisioning Service for use with the Azure IoT hub
```

Comment:
> Provision a device using the Device Provisioning Service for use with the Azure IoT hub [](start = 3, length = 88)

This is what the device client for provisioning in azure.iot.device does, this is not the domain of azure.iot.provisioning